frontend/public/components/events.jsx

```diff
@@ -195,7 +201,6 @@ export class EventsList extends React.Component {
             <ResourceListDropdown
               onChange={this.toggleSelected}
               selected={Array.from(selected)}
-              showAll
```

Comment:
This is the only place `showAll` is used. We should go ahead and remove it from `ResourceListDropdown`

Reply:
I have no problem doing that, but the showAll flag has been around since the creation of the file.  It allows for the option to have select All Resources.  If I remove the flag then I will have to remove the All Resources logic as well.

Reply:
I guess my concern here is that this PR is a bug fix for the new toolbar chips showing when everything is cleared.  Now we have changed the UX to no longer show "All" selected and now we are discussing removing unused functionality in a common component.  I worry about the expanding scope of this PR.